cmd/crane/recipes.md

```diff
@@ -79,3 +79,11 @@ crane manifest gcr.io/buildpacks/builder:v1 | jq '.config.size + ([.layers[].siz
 ```
 
 For image indexes, you can pass the `--platform` flag to `crane` to get a platform-specific image.
+
+### Add Annotations
```

Comment:
This isn't exactly a recipe, this is just how to use the tool. It's not very well defined, but I normally think of a recipe as how to use 2+ things together, like how to pipe the output of crane extract to tar, for example.

Reply:
wrong commit :/ I've removed it.